packages/labs/src/inputList.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+// import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { IProps, Keys, Utils } from "@blueprintjs/core";
+
+export interface ICoreInputListProps<D> extends IProps {
```

Comment:
I think this type param should be `T`... `D` probably isn't semantically clear enough to warrant a deviation from the naming standard (same for all the other interfaces)